src/TwbBundle/Form/View/Helper/Factory/TwbBundleFormElementFactory.php

```diff
@@ -19,4 +20,10 @@ public function createService(ServiceLocatorInterface $serviceLocator)
         $options = $serviceLocator->getServiceLocator()->get('TwbBundle\Options\ModuleOptions');
         return new TwbBundleFormElement($options);
     }
+
+    public function __invoke(ContainerInterface $container, $requestedName, array $options = NULL)

```

Comment:
For every factory:

- `createService()` method should delegate `__invoke()` method for create services, without duplicate code.
- `null` keywork must be lowercase.

Then the `__invoke()` method should use the main container like this:

```
public function __invoke(ContainerInterface $container, $requestedName, array $options = null)
{
    if ($container instanceof AbstractPluginManager) {
        $container = $container->getServiceLocator() ?: $container;
    }

    // ....
}
```